data_generators/job_description_scraper.py

```diff
@@ -18,6 +18,8 @@
 from webdriver_manager.chrome import ChromeDriverManager
 
 
+# TODO add selenium to requiremnts.txt or remove if not needed.
```

Comment:
Remove this as selenium is an avenue we cannot travel 😞 

There is an issue with selenium and the AWS service where AWS ElasticBeanstalk refreshing back to the original upload state. Reach out to Robert for more info 👍

Reply:
I removed the todo.